src/main/java/org/folio/rest/helper/ExchangeHelper.java

```diff
@@ -8,36 +8,45 @@
 import javax.money.convert.MonetaryConversions;
 
 import io.vertx.core.Context;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.folio.rest.exception.HttpException;
 import org.folio.rest.jaxrs.model.ExchangeRate;
 import org.javamoney.moneta.Money;
 
 public class ExchangeHelper extends AbstractHelper {
 
+  private static final Logger log = LogManager.getLogger();
+
   public ExchangeHelper(Context ctx) {
     super(ctx);
   }
 
   public ExchangeRate getExchangeRate(String from, String to) {
+    log.debug("getExchangeRate:: Getting exchange rate from={}, to={}", from, to);
     try {
       double exchangeRate = MonetaryConversions.getExchangeRateProvider(IDENTITY, ECB)
         .getExchangeRate(from, to)
         .getFactor()
         .doubleValue();
-
+      log.info("getExchangeRate:: Fetched exchange rate={}", exchangeRate);
```

Comment:
it will be called every time when openning invoice, I think current exchange rate will not provide some useful info for us for investigation, so propose to change to debug in both methods

Reply:
changed to debug level